### ISSUE TITLE
figure out the best way to send email notifications from server

### ISSUE THREAD

**Issue by bepetersn** [sc3/cookcountyjail#319]

**Comment by bepetersn** [comment]
Seems like the easy, pre-packaged solution to this problem is going to be Amazon's Simple Notification Service ([SNS](http://aws.amazon.com/sns/)). We can send 1000 emails a month this way. More importantly, we can actually set up subscriptions for different notifications from the server! Huzzah!

I've been fiddling with the python code to get this to work, it's not too tough.

**Comment by bepetersn** [comment]
So, I have the notification aspect totally worked out, and am currently working on using this functionality with the earlier problem we were trying to solve: sending notifications for a bad database audit result.

**Comment by bepetersn** [comment]
Reopening because I think it might make more sense to do this with the command line tool `s3cmd` instead of the `boto` library, to work with @nwinklareth's desire to use tools available from the Bash CLI.

**Comment by bepetersn** [comment]
Found this service: [zapier](https://zapier.com). 

I think it totally fulfills our needs. Their idea is to hook any set of applications together, but it also works with manual webhooks like POSTing to a URL. I set up a URL to POST to, tested it out, and was able to send myself an email from sc3.issue.notifier@gmail.com, which is an email I set up for the purpose of sending notifications. Should be able to send right to our email list. 

We get 100 emails through gmail for free, and the problems with forwarding emails are non-existent.
